### PR TITLE
rotating point around another point

### PR DESCRIPTION
**Pull request by karolselak**
Hi, I've extended rotate function a little, because rotating around another point seems to me very useful. Of course it was possible to use add and sub functions to move poing before and after rotating, but it looked terribly. I don't know how to run tests properly, but I've tested it manually. I hope that everything is right.

### CODE REVIEW

**Comment by tmcw** [comment]
Interesting, I think this is appropriate functionality but I'd prefer to keep the functions more clearly geared to a single purpose: would you mind changing this to add a new `rotateAround` method with a _required_  point to rotate around, rather than having two behaviors for the `rotate` method?

**Comment by karolselak** [comment]
Okay, I can change it as you said.

File: index.js [hunk 1]
```diff
@@ -203,8 +212,8 @@ Point.prototype = {
     },
 
     _mult: function(k) {
```
Comment (tmcw) [inline index.js:214]: Is this a bugfix? Should this be included in the rotateAround work?

Comment (karolselak) [reply index.js:214]: I'm sorry, I just add commit to my forked version and not find out that in will occur in pull request. I'll correct it.

**Comment by karolselak** [comment]
My last commit fixes mult method to work as written in docs (). In general, I use my forked version in my project, so some new changes can occures in this pull request. And now I have a doubt, is better to align code to docs or vice versa.

**Pull request closed without merging**